packages/foreman/foreman-release/foreman-release.spec

```
@@ -10,10 +10,15 @@
 
 %else
 %define repo_dir %{_sysconfdir}/yum.repos.d
+
+%if 0%{?rhel} == 7
+%define repo_dist el7
+%else
```

I wonder if instead this should be
```suggestion
%if 0%{?rhel}
%define repo_dist el%{rhel}
%else
```
to avoid similar surprises in 8.y lifetime